02.calendar/calendar.rb

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+require 'date'
+require 'optparse'
+
+options = ARGV.getopts("y:", "m:") 
+year = options["y"].to_i
+month = options["m"].to_i
+  if options["y"] != nil && options["m"] == nil
+    year = options["y"].to_i
+    month = Date.today.month
+  elsif options["y"] == nil && options["m"] != nil
```

Comment:
今回のような場合、`option["y"]`に`false`が入ってくることはないので、`option["y"] == nil`の代わりに`option["y"]`と書くことができます。

```suggestion
  elsif options["y"] && options["m"]
```